mitodl/micromasters
# Fixed various seed data command issues

#### What are the relevant tickets?

Closes #1833 

#### What's this PR do?

Fixes various issues with our seed data commands (see issue description)

#### Where should the reviewer start?

seed_data/management/commands/seed_db.py

#### How should this be manually tested?

run `./manage.py unseed_db` and `./manage.py seed_db --staff-user=<YOUR_LOGIN_USERNAME>` in succession, then check the learner search result page. Both 'Analog Learning' and 'Digital Learning' should have results with grade averages

#### Screenshots

![ss 2016-11-22 at 17 04 56](https://cloud.githubusercontent.com/assets/14932219/20543963/d5a266e4-b0d5-11e6-88d2-59c367496073.png)
